src/SpellingBee/saveState.py

```diff
@@ -0,0 +1,102 @@
+# Authors: Yah'hymbey Baruti-Bey, Jacob Lovegren
+# Course : CSCI 420
+# Modified Date: 2/2/2023
+# State structure for puzzles
+
+import sqlite3
+import generateSubset
+
+class Puzzle:
+    
+    def __init__(self, keyLett, pangram):
+        self.keyLett = keyLett
+        self.pangram = pangram
+        self.score = 0
+        self.maxScore = 0
+        self.foundWordList = []
+        self.rank = ' '
+        
+    def showKeyLetter(self):
+        return self.keyLett
+    
+    def showUniqueLetters(self):
```

Comment:
to reiterate comment on pangram ambiguity, would be consistent with fucntion naming here